js/alerts.js

```diff
@@ -385,20 +385,21 @@ function performAction() {
         else if (action == "Add Bug") {
             var BugID = prompt("Please enter Bug ID");
             console.log('BUG ID-'+BugID);
+            var alertids = '';
+            for (id in checkedIds) {
+                alertids = alertids + checkedIds[id] + ",";
+            }
```

Comment:
here you could use the .join() method (http://www.w3schools.com/jsref/jsref_join.asp)